app/controllers/api/logins_controller.rb

```diff
@@ -3,24 +3,30 @@ class Api::LoginsController < Api::ApplicationController
   before_action :create_debate_auth_token
 
   def create
-    form = ::MobileUserForm.new(::MobileUser.new)
-    if form.validate(user_params.merge(auth_token: @auth_token))
-      form.save
+    user = MobileUser.new(user_params)
+    if user.save
       render json: { auth_token: @auth_token.value, debate_closed: @debate.closed? }
     else
-      render json: { error: form.error_messages }, status: :bad_request
+      render json: { error: user_errors(user) }, status: :bad_request
     end
   end
 
   private
 
+  def user_errors(user)
```

Comment:
Zakładając, że wprowadzimy inne zmiany, to nie ma chyba sensu zwracać tutaj wszystkich błędów walidacji usera? Jedyne co może być nie tak, to że brakuje `auth_token`, więc można po prostu zwrócić `"Invalid auth token"`.